### ISSUE TITLE
Afwall error when applying rules

### ISSUE THREAD

**Issue by kohanla** [ukanth/afwall#1305]
I tested this on various devices (A11 and A12). I am not able to reproduce this consistently, but it happens (very) frequently.

AFwall is not able to apply rules without an error "in the first round". I have to repeat the step many times to get rules applied.

But there is one way to fix the issue 100%. I just need to switch from "IPv6 support" to "Only control IPv6" mode (and block all chains). After that applying rules never errors out.

**Comment by CircuitSerialKiller** [comment]
Same issue here with 3.5.3 and OnePlus 6T and 8T. The issue here seems to still linger with IPv6:
https://github.com/ukanth/afwall/issues/1206

**Comment by vsatmydynipnet** [comment]
I can confirm this bug on LineageOS 19.1 / Fairphone 3+

It does not happen on LineageOS 18 / Samsung A5/2017

It looke like if happens very often when changing network connections, e.g. WLAN to LTE and back. 

Needs reboot of device to be ok again. On book and some time later it loads including custom rules without any error.

**Comment by upbox-org** [comment]
Do you checked the "Add Delay" option? 
This fixed the error for me at least. No need to disable IPv6 support here.

**Comment by kohanla** [comment]
Yes, I tried that option (I probably tried everything :). Did not help unfortunately.

**Comment by breversa** [comment]
I have a similar behaviour when I install a new app, then tap the notification to add that app to whitelist: applying the rules fails 100% of the time.

Killing AFwall+ (with long-press on "back", in my ROMs), starting it again then applying the rules then works 100%.

**Comment by iromeister** [comment]
Same here using /e/OS on FP4

**Comment by akronym0** [comment]
I had the same problem, but since Android 14 it works fine.
